### DISCUSSION TITLE
Any examples?

### DISCUSSION THREAD

**Discussion by IgorAlexey** [crud89/LiteFX#80] [Q&A] [answered]
Hi, I found this library which supports the features I wanted for my project.
Are there any 3D examples or samples that can introduce me to the key concepts and usage of the library? 😃

**Comment by crud89** [comment, marked as answer]
Sorry for the late response. I am currently working on a larger project, but for the time being you can take a look at the samples, that are included (They are in the `src/samples/` directory). Please note however, that this is still not a production-ready library, so there may be bugs or unexpected behavior. So if you find any, feel free to open an issue. 🙂
